NosTests/Service/MockFeatureFlags.swift

```diff
@@ -1,8 +1,16 @@
 /// A set of feature flag values used for testing that can be customized.
 class MockFeatureFlags: FeatureFlags {
-    var newMediaDisplayEnabled = false
+    /// A dictionary to store the mock values for feature flags.
```

Comment:
Same suggestion here about mentioning the type in the doc comment.

Reply:
Thanks for these feedback and suggestion @bryanmontz. I will make this changes now.